lib/pymedphys/_streamlit/apps/wlutz/_dbf.py

```diff
@@ -0,0 +1,136 @@
+# Copyright (C) 2020 Cancer Care Associates
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pymedphys._imports import pandas as pd
+from pymedphys._imports import streamlit as st
+
+from pymedphys._streamlit.utilities import dbf
+
+
+DBF_DATABASE_LOADING_CONFIG = {
+    "frame": {
+        "filename": "FRAME.dbf",
+        "columns_to_keep": ["DBID", "PIMG_DBID", "DELTA_MS"],
+        "column_rename_map": {"DBID": "FRAME_DBID"},
+    },
+    "patimg": {
+        "filename": "PATIMG.dbf",
+        "columns_to_keep": [
+            "DBID",
+            "DICOM_UID",
+            "IMG_DATE",
+            "IMG_TIME",
+            "PORT_DBID",
+            "ORG_DTL",
+        ],
+        "column_rename_map": {"DBID": "PIMG_DBID", "ORG_DTL": "machine_id"},
+    },
+    "port": {
+        "filename": "PORT.dbf",
+        "columns_to_keep": ["DBID", "TRT_DBID", "ID"],
+        "column_rename_map": {"DBID": "PORT_DBID", "ID": "port"},
+    },
+    "trtmnt": {
+        "filename": "TRTMNT.dbf",
+        "columns_to_keep": ["DBID", "PAT_DBID", "ID"],
+        "column_rename_map": {"DBID": "TRT_DBID", "ID": "treatment"},
+    },
+    "patient": {
+        "filename": "PATIENT.dbf",
+        "columns_to_keep": ["DBID", "ID", "LAST_NAME", "FIRST_NAME"],
+        "column_rename_map": {"DBID": "PAT_DBID", "ID": "patient_id"},
+    },
+}
+
+
+def loading_and_merging_dbfs(database_directory, refresh_cache):
```

Comment:
load_and_merge_dbfs would be a better name.
needs docstring (it's a public method even if it's in a private module, whoever maintains the module should know what it is)